### DISCUSSION TITLE
Running on Azure app service gives Minio errors

### DISCUSSION THREAD

**Discussion by craftzneko** [outline/outline#5588] [Self Hosting]
I am having issues getting docker compose.yaml to work in Azure app service. The issue seems to be realted to the minio image. Using the supplied docker compose.yaml file resultrs in 

ERROR server': -c: line 0: unexpected EOF while looking for matching `''

The part in the docker compose is

    entrypoint: sh
    command: -c 'minio server'
    deploy:
      restart_policy:
        condition: on-failure
        
I have tried changing it to some recomended versions elshewhere but get issues with that. FInally changing it to

    entrypoint: sh
    command: ["-c", "minio server /data"]
    deploy:
      restart_policy:

seems to give me the best way forward but also errors with ERROR Unable to use the drive /data: drive access denied: Invalid arguments specified

Hoping someone has tried this on Azure app service before and knows if i am missing anything here

**Comment by tommoor** [comment]
I'd suggest asking your question on the minio project. With Azure you can just use their cloud storage with S3-compatibility rather than using Minio.
